src/i_video.h

```diff
@@ -82,6 +82,10 @@ void I_StartTic (void);
 
 void I_EnableLoadingDisk(int xoffs, int yoffs);
 
+// Switch between fullscreen and windowed mode
+
+void I_ToggleFullScreen();
```

Comment:
This makes for a strange API function. Can you just move everything into i_video.c, since that's part of the input processing chain anyway? ie. intercept the fullscreen keypress before in I_GetEvent before it's even passed to the code in i_input.c.